statchatui/src/Stats/Stats.jsx

```diff
@@ -1,25 +1,29 @@
 import React from "react";
 
-const Stats = ({ statData }) => {
+const Stats = ({ statData, data }) => {
```

Comment:
This looks good, but would have made more sense in its own PR (i might have misunderstood you on the call). A quick test for this is to see whether it's related to the other changes in your pull request. 

And just to clarify, this isn't quite the same thing as form validation, so there's a bit more we can do in that area :)